.github/actions/generate-builder/action.yml

```diff
@@ -62,7 +51,5 @@ runs:
       run: ./.github/actions/generate-builder/generate-builder.sh
 
     - name: Compute sha256 of builder
-      uses: slsa-framework/slsa-github-generator/.github/actions/compute-sha256@1d646d70aeba1516af69fb0ef48206580122449b
+      uses: slsa-framework/slsa-github-generator/.github/actions/compute-sha256@9c56e223d2486617188b6a11ed385ac1a94d77c0
```

Comment:
I'm afraid this won't work because this commit digest doesn't exist in the repo.

```shell
$ git log 9c56e223d2486617188b6a11ed385ac1a94d77c0
fatal: bad object 9c56e223d2486617188b6a11ed385ac1a94d77c0
```

Reply:
This looks like an GH Issue because this commit exists in our fork (https://github.com/Root-DE/slsa-github-generator/commit/9c56e223d2486617188b6a11ed385ac1a94d77c0) but you can't find it even when using the general GH search. To enable contributions it seems like we would need a branch (This is only necessary due to the necessaty of hard coded commits hashs). Can you create a branch for us? We would then merge our stuff into this one for testing.

Reply:
You're right. It does exist in your fork, but here you are referencing the main repository (`slsa-framework/slsa-github-generator`) so this action call will fail because that commit digest does not exist in the `slsa-framework/slsa-github-generator` repo. It only exists in the `Root-DE/slsa-github-generator` repo.

If you need to update this sha to incorporate some changes then you need get a PR merged in the main repository and then create a new PR reference that previous merge commit digest.

BTW, what was the reason you needed to update these digests?

Reply:
I see, we updated these digest due to the changes made in compute-sha256